pkg/volume/plugins.go

```diff
@@ -55,6 +55,10 @@ type VolumeOptions struct {
 	ClusterName string
 	// Tags to attach to the real volume in the cloud provider - e.g. AWS EBS
 	CloudTags *map[string]string
+	// ProvisioningOptions are additional options for the provisioner, e.g.
+	// to create a volume of certain QoS characteristics. These options are
+	// opaque to anything but appropriate provisioner plugin.
+	ProvisioningOptions string
```

Comment:
We were originally going to use `map[string]string`.  @jsafrane thought small structs with the provisioner's options made it easy to parse with `json.Unmarshall(str, &ebsProvisionerStruct{})`.  Since the string and all the params it represents is opaque to the system, I agreed with him.

Turning this into a map is not difficult, but does it expose the plugin's internals too much?